### PR TITLE
SD-1612 Define how to populate data

### PR DESCRIPTION
**Pull request by amagdas**
Phase 1: Add data initialization files and a script to bootstrap the mongo database.
In another pull request we will import editorial content and ingest sources.

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: server/superdesk/data_initialization/import.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (petrjasek) [inline server/superdesk/data_initialization/import.sh]: would it be possible to make this manage command?

Comment (amagdas) [inline server/superdesk/data_initialization/import.sh]: It is just that it's a bit more work. This is on the plan just that I added this if we need a way to bootstrap a new env quickly.

**Comment by amagdas** [comment]
@superdesk/developers @sivakuna-aap @marwoodandrew @mdhaman @akintolga

**Comment by mdhaman** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Pull request merged**